docs/configuration/git-providers/gitlab-enterprise.md

```diff
@@ -1,24 +1,34 @@
 # GitLab Enterprise
 
-Set your configuration values for your GitLab instance on the `values.yaml` file.
-**Please note that you must go to `http://codacy.example.com/admin/integrations`, select the desired provider and `Test & Save` your configuration for it to be applied.**
-
 ## GitLab Application
 
-To integrate with GitLab we use a GitLab Application.
+Follow the instructions below to set up the Codacy integration with GitLab Cloud:
 
-Create an application pointing to your local Codacy deployment URL.
+1.  Create a new application in `<gitlabEnterprise url>/profile/applications`, where `<gitlabEnterprise url>` is the url of your own GitLab Enterprise instance, pointing to your local Codacy deployment URL with `api`, `read user` and `read repository` scopes.
 
-You'll need to add the following 'Redirect URI'. Make sure to update your protocol to use either http or https and your domain name as well. Keep in mind this field is case sensitive.
+    You'll need to add the following 'Redirect URI'. Make sure to update your protocol to use either http or https and your hostname as well. Keep in mind this field is case sensitive.
```

Comment:
```suggestion
    You'll need to add the following "Redirect URI", making sure to update both your protocol to use either HTTP or HTTPS and your hostname as well. Keep in mind that this field is case sensitive.
```

Reply:
The user must enter all the URIs, right? Are they entered one in each field, or all of them in the same field? :thinking:

Reply:
All of them are entered in the same field